include/alpaka/queue/QueueCpuBlocking.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include <alpaka/core/Unused.hpp>
 #include <alpaka/dev/DevCpu.hpp>
-#include <alpaka/queue/cpu/ICpuQueue.hpp>
+#include <alpaka/queue/QueueGenericBlocking.hpp>
```

Comment:
all expect this include should be removed

Reply:
I cannot follow, can you please clarify?

This file defines an alias for QueueGenericBlocking<DevCpu> and thus needs the definition of that template.

Do you mean this file should be removed and thus the alias? This would entail lots of changes elsewhere, i.e. QueueCpuBlocking -> QueueGenericBlocking<DevCpu>

Reply:
I think @psychocoderHPC  meant that you can remove nearly all other includes in this file because you only create a typedef for a speialization of `QueueGenericBlocking`. So you only need its header and the header of the template parameter.
This is also true for the other typedef-only headers.

Reply:
That does make a lot of sense, thanks. Done.